cypress/support/helper/http.js

```diff
@@ -0,0 +1,11 @@
+const axios = require('axios');
```

Comment:
Why do you decided to use an own axios Implementation instead of Cypress' Ines? E.g. `cy.request`

Reply:
They don't support async / await / promise. Also usage of cy commands in cy commands throws a warning 😓

Reply:
Ah, I got it. Had that in `sw-cypress` as well but I tried to keep the Cypress tools as far as possible due to consistency and smaller learning curve for other QA engineers. May change that in the future as well. 

However, as our current Axios version causes security alerts and you're currently working at it anyway, could you update the dependency to the latest version? Would be wonderful :heart:

Reply:
For further information for that command issue see https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Commands-Are-Not-Promises

Reply:
Its update to date. Security check only considerns the master branch :)